corehq/apps/analytics/tasks.py

```diff
@@ -144,6 +151,7 @@ def _get_client_ip(meta):
     return ip
 
 
+@increment_metric('sent_form_to_hubspot')
```

Comment:
Instead of wrapping the whole function you could do something like this:
```
@count_by_response_code('commcare.sent_form_to_hubspot')
def _post_to_url(url, data):
    return requests.post(url, data)
```

Doing it this way would capture the status code as well.